core/src/test/java/io/grpc/internal/ManagedChannelImplIdlenessTest.java

```diff
@@ -205,15 +204,14 @@ public void enterIdleModeAfterForceExit() throws Exception {
     newTransports.drainTo(transports);
     assertEquals(addressGroupList.size(), transports.size());
 
-    InterimTransport<ClientTransport> interimTransport = channel.tm.createInterimTransport();
+    channel.tm.createInterimTransport();
```

Comment:
Looking at InterimTransport, wouldn't be better to keep the ref and close it at the end of this method?